modules/ROOT/pages/get-started-prepare.adoc

```diff
@@ -40,10 +40,15 @@ Here's what you need in order to install Sync Gateway:
 
 Once you have all that covered ... go {get-started-install--xref} {sgw-t}.
 
+== Cluster Cluster Requirements
```

Comment:
This must be a typo. "**Cluster** Cluster Requirements" should be "Couchbase Cluster Requirements".

Please also see below comments. The place of this chapter (and the level of chapter) seems misleading to readers.